Sources/SwiftSyntax/SyntaxText.swift

```diff
@@ -10,14 +10,6 @@
 //
 //===----------------------------------------------------------------------===//
 
-#if !SWIFT_SYNTAX_ALWAYS_SINGLE_THREADED
-#if canImport(Darwin)
-@_implementationOnly import Darwin
-#elseif canImport(Glibc)
-@_implementationOnly import Glibc
-#endif
-#endif
```

Comment:
Forgot to remove this. This should resolve "ambiguous `memcmp`" errors. But may affects Linkage tests. checking locally.

Reply:
Updated the test.